apps/frontend/src/modules/activity/activity-logs-api.ts

```diff
@@ -29,6 +29,7 @@ class ActivityLogApi extends BaseHTTPApi {
 				hasAuth: true,
 				method: "GET",
 				query: {
+					contributorName: String(query.contributorName),
```

Comment:
Isn't `query.contibutorName` already a string? If it possibly undefined, then this is not very good idea to wrap it with String constructor, as in this case we'll try to fined someone with name "undefined"
```suggestion
					contributorName: query.contributorName,
```

Reply:
Oh.. thanks, have missed it